packages/filebrowser/src/listing.ts

```diff
@@ -2329,4 +2495,12 @@ namespace Private {
       container.className = HEADER_ITEM_ICON_CLASS;
     }
   }
+
+  /**
+   * An event handler that just prevents the event's default action from
+   * happening (such as checking a checkbox after a click).
+   */
+  export function preventDefault(event: Event): void {
```

Comment:
Do we need this extra function? Looks like it is used in one place only for now?

Reply:
No it's not needed. Probably did it out of old habit, not wanting to create a multiple function instances when the event callbacks could all point to the same function in memory.

Reply:
I'm guessing that's probably a premature optimization these days